.travis.yml

```diff
@@ -32,6 +32,9 @@ services:
 
 sudo: false
 
+before_install:
+  - composer config -g github-oauth.github.com $CI_USER_TOKEN
```

Comment:
mmm, actually it works but you are publishing the token:

![screen shot 2015-12-21 at 12 20 40](https://cloud.githubusercontent.com/assets/1375475/11929644/4ffb9166-a7dd-11e5-94e0-025f044a0c1f.png)